build/common.props

```diff
@@ -4,6 +4,7 @@
   <Import Project="version.props" />
 
   <PropertyGroup>
+    <LangVersion>preview</LangVersion>
```

Comment:
Looks like the CI doesn't like this for some reason.

Reply:
Yeah, I guess that's because it requires a 3.0 SDK 😢